galite-core/src/main/kotlin/org/kopi/galite/util/Fax.kt

```diff
@@ -0,0 +1,770 @@
+   * Verzeichnisses als String zurueck.
+   */
+  fun infoS(what: String): String {
+    val pstr: String // String für port
+    val iaddr: ByteArray // byte-array für Internetadresse
+    val recsrv: RecvServ // server-thread-object
+
+    // Thread erzeugen
+    recsrv = RecvServ(debug)
+    iaddr = getInetAddr()
+    pstr = makePORT(iaddr, recsrv.port)
+    print("PORT $pstr\n")
+    clntOut.flush()
+    check(readLine())
+    print("LIST $what\n")
+    clntOut.flush()
+
+    // Auf Beendigung des Threads warten
+    try {
+      recsrv.join()
+    } catch (e: InterruptedException) {
+    }
+    if (typeOfThread != NONE) {
+      throw PROTOException(errText, LOST_CONNECTION)
+    }
+    if (check(readLine()) == ABOUT_TO_OPEN_DATACON) {
+      check(readLine())
+    } else {
+      throw PROTOException("Fax.infoS: No Data from Faxserver", 1)
+    }
+    var cnt: Int // Anzahl der Zeichen zaehlen
+    cnt = 0
+    while (cnt < recsrv.data.size) {
+      if (recsrv.data[cnt].toInt() == 0) {
+        break
+      }
+      cnt++
+    }
+
+    //hibyte - the top 8 bits of each 16-bit Unicode character
+    return String(recsrv.data, 0, cnt)
+  }
+
+  fun command(what: String): String {
+    /*
+     * Sendet ein Kommando an den Faxserver und
+     * gibt die Antwort zurueck oder erzeugt ein
+     * Ausnahmeobjekt
+     */
+    val response = StringBuffer()
+    var line: String
+    print("""
+  $what
+  
+  """.trimIndent())
+    clntOut.flush()
+    val erg = check(readLine())
+
+    // SYSTEM_STATUS und HELP_MESSAGE sind laenger als
+    // eine Zeile
+    if (erg == SYSTEM_STATUS || erg == HELP_MESSAGE) {
+      while (true) {
+        line = readLine()
+        if (check(line) == erg) {
+          break
+        }
+        response.append("""
+  $line
+  
+  """.trimIndent())
+      }
+    }
+    return response.toString()
+  }
+
+  private fun setNewJob(number: String, user: String, id: String) {
+    // number check:
+    var number = number
+    var user = user
+    number = checkNumber(number)
+    user = DEFAULT_USER
+    Utils.log("Fax", "NEW JOB:$id / user: $user")
+
+    // Jobparameter einstellen
+    command("JNEW")
+    command("JPARM FROMUSER \"$user\"")
+    command("JPARM LASTTIME 145959")
+    command("JPARM MAXDIALS 3")
+    command("JPARM MAXTRIES 3")
+    command("JPARM SCHEDPRI 127")
+    command("JPARM DIALSTRING \"$number\"")
+    command("JPARM NOTIFYADDR \"$user\"")
+    command("JPARM JOBINFO \"$id\"")
+    command("JPARM VRES 196")
+    command("JPARM PAGEWIDTH " + 209)
+    command("JPARM PAGELENGTH " + 296)
+    command("JPARM NOTIFY \"NONE\"") //1:mail when done
+    command("JPARM PAGECHOP \"default\"")
+    command("JPARM CHOPTHRESHOLD 3")
+  }
+
+  private fun checkNumber(number: String): String {
+    var newNumber = ""
+    for (i in 0 until number.length) {
+      if (number[i] >= '0' && number[i] <= '9') {
+        newNumber += number[i]
+      }
+    }
+    return newNumber
+  }
```

Comment:
Inline it with : 
    = buildString {
      number.filter { it in '0'..'9' }.forEach { append(it) }
    }